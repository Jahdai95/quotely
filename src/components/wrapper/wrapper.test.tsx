import { render } from "@testing-library/react";
import { describe, test, it, expect } from "vitest";
import Wrapper from "./wrapper";
import { QuotesProvider } from "../../providers/quotes.provider";

describe("Should render Widget component", () => {
  const getRender = () => {
    return render(
      <QuotesProvider>
        <Wrapper>
          <h1>No dejes para ma√±ana lo que puedes hacer hoy</h1>
        </Wrapper>
      </QuotesProvider>
    );
  };

  it("Should macth snapshot", () => {
    const { container } = getRender();
    expect(container).toMatchSnapshot();
  });
});
