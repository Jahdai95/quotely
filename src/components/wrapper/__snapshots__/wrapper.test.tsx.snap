// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Should render Widget component > Should macth snapshot 1`] = `
<div>
  <div
    class="background"
  >
    <div
      class="container"
    >
      <div
        class="main"
      >
        <h1>
          No dejes para ma√±ana lo que puedes hacer hoy
        </h1>
      </div>
    </div>
  </div>
</div>
`;
